code-challenges.js

```diff
@@ -9,9 +9,46 @@
 
 // Create the code and test each of the variables provided. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 35
-// var temp = 350
-// var temp = 212
+// I'm renaming the variable 'temp' to 'temperature' for clarity and readability.
+var temperature = 35
+// var temperature = 350
+// var temperature = 212
+
+// I can write a basic 'if-else' statement that will compare the current value with the number 212, and return the expected output based on wether the current value is greater or less than 212. This works. There's probably a cleaner way though.
+
+if (temperature == 212) {
+    console.log("212 is boiling point")
+} else if (temperature == 350) {
+    console.log("350 is above boiling point")
+} else if (temperature == 35) {
+    console.log("35 is below boiling point")
+} else {
+    console.log("Those are your only choices")
+}
```

Comment:
I like this, but what would make this more dynamic is if you called on the >, < operators. 
```javascript
if (temperature == 212) {
    console.log("input is boiling point")
} else if (temperature > 212) {
    console.log("input is above boiling point")
} else if (temperature < 212) {
    console.log("input is below boiling point")
} else {
    console.log("Those are your only choices")
}
```